docs/docs/3-features-modules/10-redirects/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Redirects
+---
+
+## Custom redirect targets
+
+Custom redirect targets (e.g., a news site) are supported via the `customTargets`-option.
+The option expects a block:
+
+```ts title=app.module.ts
+RedirectsModule.register({
+    // highlight-next-line
+    customTargets: { news: NewsLinkBlock },
+});
+```
+
+```tsx title=App.tsx
+const RedirectsPage = createRedirectsPage({
+    // highlight-next-line
+    customTargets: { news: NewsLinkBlock },
+});
+```
+
+A custom target URL service is required to resolve the target URLs:
+
+```ts title=my-redirect-target-url.service.ts
+@Injectable({ scope: Scope.REQUEST })
+export class MyRedirectTargetUrlService implements RedirectTargetUrlServiceInterface {
+    constructor() {}
+
+    async resolveTargetUrl(
+        target: ExtractBlockData<RedirectsLinkBlock>["attachedBlocks"][number],
+    ): Promise<string | undefined> {
+        if (target.type === "internal") {
+            const targetPageId = (target.props as ExtractBlockData<typeof InternalLinkBlock>)
+                .targetPageId;
+
+            if (targetPageId) {
+                const targetPageNode = await this.pageTreeReadApi.getNode(targetPageId);
+
+                if (!targetPageNode) {
+                    return undefined;
+                }
+
+                return this.pageTreeReadApi.nodePath(targetPageNode);
```

Comment:
Shouldn't this example also include the baseUrl to support absolute URLs?

Reply:
I wanted to keep the example in the docs as slim as possible, but I can add that.

Reply:
Added a full example in a collapsible: 10ed0e428333c991a1c8990c5a9c8002b91eedc3